src/Model/Componentfactory/include/Componentfactory.hpp

```diff
@@ -157,11 +205,27 @@ namespace Model::Componentfactory {
     std::unique_ptr<Network::Node>
     make_instance(nlohmann::json const &topology) const override {
 
-      validate_component_json(topology, ConcreteNode::get_schema());
+      validation::validate_json(topology, this->get_schema(true));
```

Comment:
NodeType::get_schema() adds stuff to the schema from ConcreteNode so get_schema from concreteNode should not be used directly

Reply:
makes sense.